CHANGELOG.md

```diff
@@ -3,7 +3,13 @@
 ## 17.0.0 - unreleased
 This release includes several breaking changes. See the [migration guide](https://github.com/stripe/stripe-android/blob/master/MIGRATING.md) for more details.
 
-* [3820](https://github.com/stripe/stripe-android/pull/3820) Upgrade Kotlin to `1.5.10` and Coroutines to `1.5.0`
+* [3820](https://github.com/stripe/stripe-android/pull/3820) Upgrade Kotlin to `1.5.10`
+* [3899](https://github.com/stripe/stripe-android/pull/3899) Introduce `GooglePayLauncher` and `GooglePayPaymentMethodLauncher`
+    * Drop-in classes that simplify integrating Google Pay
+* [3944](https://github.com/stripe/stripe-android/pull/3944) Introduce `GooglePayLauncherContract` and `GooglePayPaymentMethodLauncherContract`
+    * `ActivityResultContract` classes that enable integrating `GooglePayLauncher` and `GooglePayPaymentMethodLauncher` in Jetpack Compose
+* [3951](https://github.com/stripe/stripe-android/pull/3951) Upgrade material-components to `1.4.0`
+* [3976](https://github.com/stripe/stripe-android/pull/3976) Upgrade Kotlin Coroutines to `1.5.1`
 
 ## 16.10.2 - 2021-07-02
 * [3811](https://github.com/stripe/stripe-android/pull/3811) Fix `CountryCode` parceling
```

Comment:
I was not sure about these:
55d9b07317176291e45090f5d4ad8fe5ee18937a (update-bug-request) Simple refactor: move returnUrl to `WebIntentAuthenticator` and clean up `IntentAuthenticator` interface (#3914)
35d7c9678b975fd24914bb52ec425bf81c0be0d7 Create Stripe3ds2TransactionActivity and related classes (#3875)
738c67adbab159cb43493bf7b9cfe7c69858f750 Remove PaymentIntent.nextAction (#3883)
adab8fb4b2a7f129a935bf3aadbc3c558255f3f6 apiDump for Google Pay package renaming (#3827)
4b468bb9361cedd4be917025e3a8d1f6b9fab1f1 Remove Card.loggingTokens (#3819)